src/traces/contour/attributes.js

```diff
@@ -155,6 +183,53 @@ module.exports = extendFlat({
                 'https://github.com/d3/d3-format/blob/master/README.md#locale_format.'
             ].join(' ')
         },
+        operation: {
+            valType: 'enumerated',
+            values: [].concat(COMPARISON_OPS2).concat(INTERVAL_OPS),
+            role: 'info',
+            dflt: '=',
+            editType: 'calc',
+            description: [
+                'Sets the filter operation.',
+
+                '*=* keeps items equal to `value`',
+
+                '*<* and *<=* keep items less than `value`',
+
+                '*>* and *>=* keep items greater than `value`',
+
+                '*[]*, *()*, *[)*, and *(]* keep items inside `value[0]` to `value[1]`',
+
+                '*][*, *)(*, *](*, *)[* keep items outside `value[0]` to value[1]`',
+
+                'Open vs. closed intervals make no difference to constraint display, but',
+                'all versions are allowed for consistency with filter transforms.'
+            ].join(' ')
+        },
+        value: {
+            valType: 'any',
+            dflt: 0,
+            role: 'info',
+            editType: 'calc',
+            description: [
+                'Sets the value or values by which to filter by.',
+
+                'Values are expected to be in the same type as the data linked',
+                'to *target*.',
```

Comment:
Hmm. I think this was copied from `transforms/filter.js`, `target` makes no sense here.

Reply:
good catch - https://github.com/plotly/plotly.js/pull/2270/commits/7c7a01135b0319fa536d7520b6fc33f8275d27c5